cosmos/injective.json

```diff
@@ -70,6 +70,20 @@
       "coinDecimals": 6,
       "coinMinimalDenom": "factory/inj127l5a2wmkyvucxdlupqyac3y0v6wqfhq03ka64/qunt",
       "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/injective/factory/inj127l5a2wmkyvucxdlupqyac3y0v6wqfhq03ka64/QUNT.png"
+    },
+    {
+      "coinDenom": "GLTO",
+      "coinMinimalDenom": "peggy0xd73175f9eb15eee81745d367ae59309Ca2ceb5e2",
```

Comment:
Is it a CW token or factory token?

Reply:
This is Peggy asset which is ETH asset bridged to INJ via their bridge.